Documentation/devel/stage1-implementors-guide.md

```diff
@@ -118,3 +118,22 @@ Examples
     ]
 }
 ```
+
+Filesystem Layout Assumptions
+-----------------------------
+
+### opt/stage2
+
+As mentioned above, in rkt, the apps will be executed during stage2.
+rkt expects the directory path `opt/stage2` to exist under the stage1 rootfs.
```

Comment:
Actually, we probably still have to write something about `/opt/stage2` directories being created by rkt, so stage1 implementation shouldn't have any file named `stage2` in `/opt` directory. Or something like that.